Sources/FeedbacksKit/Model/FeedbackFormData.swift

```diff
@@ -1,12 +1,24 @@
+import UIKit
+
 public struct FeedbackFormData {
     let email: String
     let message: String
 
+	let deviceName: String
+	let systemNameAndVersion: String
+	let appVersion: String
+	let language: String
+
     public init(
         email: String,
         message: String
     ) {
         self.email = email
         self.message = message
+
+		deviceName = UIDevice.current.name
```

Comment:
I don't think that's useful.
- For devices before iOS16 this will return the user defined name like _"Lawmaestro's iPhone 15"_. I don't want that in order to preserve user's privacy.
- For devices after iOS16, this will only return a generic name, then it's useless considering you also added the `systemName`.

[Documentation](https://developer.apple.com/documentation/uikit/uidevice/1620015-name)

Reply:
I've updated this to what I actually originally intended it to be which is just a device identifier which can be cross referred to find the device model/info. This of course doesn't have an user PI related info in either ;)